AcadeMe/views.py

```diff
@@ -7,6 +7,7 @@ def app_layout(request):
 
 
 def homePage(request):
+    print(request.user)
```

Comment:
do we need this print?

Reply:
no hell we dont!
ill delete it now sorry mate